eng/pipelines/steps/compound-esrp-code-signing-step.yml

```diff
@@ -5,20 +5,15 @@
 #################################################################################
 
 parameters:
-    - name: ESRPConnectedServiceName
-      type: string
-
     - name: appRegistrationClientId
```

Comment:
Can you rename these too for consistency?

Reply:
There's no real reason to, since at this point all the parameters have to do with signing (it is the _signing_ task after all :) )